src/Symfony/Component/Form/Extension/Core/Type/DateTimeType.php

```diff
@@ -278,6 +278,28 @@ public function configureOptions(OptionsResolver $resolver)
             'text',
             'choice',
         ));
+
+        $resolver->setNormalizer('date_format', function (Options $options, $dateFormat) {
+            if (null !== $dateFormat && 'single_text' === $options['widget']) {
+                @trigger_error(sprintf('Using the "date_format" option of the %s when the "widget" option is set to "single_text" is deprecated since Symfony 4.2.', self::class), E_USER_DEPRECATED);
```

Comment:
how will this behave in v5? throw an exception? silently ignoring? we should add a hint to users and for the next maintainer that will update the code when preparing v5